openshift-extension/src/test/java/io/quarkus/ts/openshift/extension/OpenShiftClientResource.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.ts.openshift.extension;
+
+import io.fabric8.openshift.client.DefaultOpenShiftClient;
+import io.fabric8.openshift.client.OpenShiftClient;
+import org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource;
+
+final class OpenShiftClientResource implements CloseableResource {
```

Comment:
You don't want this, this is only useful for JUnit 5 extensions. Just create `DefaultOpenShiftClient` directly.